ci/cudf_pandas_scripts/pandas-tests/diff.sh

```diff
@@ -8,17 +8,22 @@
 
 # Hard-coded needs to match the version deduced by rapids-upload-artifacts-dir
 MAIN_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py310.main-results.json
-PR_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py310.pr-results.json
-aws s3 cp $MAIN_ARTIFACT main-results.json
+PR_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py39.pr-results.json
```

Comment:
It would be better if these versions were not hard coded if possible.

If that can't be accomplished easily, please file an issue for it but extract the values to variables defined prominently in the beginning of the file.

Reply:
I don't think that would still work here, the actual job that generate's the json files is different from the job that runs this file, so we wouldn't be able to utilize the machine's versions here.